client/src/components/Navbar.js

```diff
@@ -53,6 +54,16 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Navbar() {
     const classes = useStyles()
+    const [iconURL, setIconURL] = useState(process.env.PUBLIC_URL + 'images/avatar-1.png')
+
+    useEffect(() => {
+        
+        getProfile((data) => {
+            setIconURL(data.icon)
+        }, (error) => {
+            console.log(error)
+        })
+    })
```

Comment:
i think you need a dependency here for the useEffect

Reply:
This will be replaced once we use UserContext.